server/enterprise.go

```diff
@@ -59,6 +59,11 @@ type EETriggerInfo struct {
 func (s *Server) getPRInfo(ctx context.Context, pr *model.PullRequest) (info *EETriggerInfo, err error) {
 	pullRequest, _, err := s.GithubClient.PullRequests.Get(ctx, s.Config.Org, pr.RepoName, pr.Number)
 	if err != nil {
+		mlog.Debug("Error trying to get pr",
```

Comment:
I would prefer it to be wrapped with a context string here, and then logged at the call site.